test/unit.js

```diff
@@ -216,6 +465,10 @@ test('linux - musl version detected via child process', async (t) => {
 
   t.is(await libc.version(), '1.2.3');
   t.is(libc.versionSync(), '1.2.3');
+
+  // calling twice to check the cache
```

Comment:
Is this verifying that the cache was used? Unsure we need this if not.

Reply:
If not, the coverage will not be 100%.